src/components/Header/index.module.css

```diff
@@ -0,0 +1,8 @@
+.header {
```

Comment:
The Header component is styled incorrectly.

On the prototype you can see that there aren't any top and left paddings:

<img width="438" alt="Screenshot 2020-11-09 at 12 47 50" src="https://user-images.githubusercontent.com/16220722/98532402-51509980-228a-11eb-84ec-4856bbd2769b.png">

On the other hand, this implementation has the extra top padding:

![Screenshot 2020-11-09 at 12 52 17](https://user-images.githubusercontent.com/16220722/98532616-9c6aac80-228a-11eb-8ee3-e4828f4eab21.png)

The solution here is to remove all rules from `Header/index.module.css`.

Reply:
I have removed styles